locales/tr/tr.json

```diff
@@ -1,618 +1,618 @@
     "Honduras": "Honduras",
     "Hong Kong": "Hong Kong",
-    "Hungary": "Hungary",
-    "I agree to the :terms_of_service and :privacy_policy": "I agree to the :terms_of_service and :privacy_policy",
-    "Iceland": "Iceland",
-    "ID": "ID",
-    "If necessary, you may log out of all of your other browser sessions across all of your devices. Some of your recent sessions are listed below; however, this list may not be exhaustive. If you feel your account has been compromised, you should also update your password.": "If necessary, you may log out of all of your other browser sessions across all of your devices. Some of your recent sessions are listed below; however, this list may not be exhaustive. If you feel your account has been compromised, you should also update your password.",
+    "Hungary": "Macaristan",
+    "I agree to the :terms_of_service and :privacy_policy": ":terms_of_service ve :privacy_policy'yi kabul ediyorum",
+    "Iceland": "İzlanda",
+    "ID": "KİMLİK",
+    "If necessary, you may log out of all of your other browser sessions across all of your devices. Some of your recent sessions are listed below; however, this list may not be exhaustive. If you feel your account has been compromised, you should also update your password.": "Gerekirse, tüm cihazlarınızda diğer tüm tarayıcı oturumlarınızdan çıkış yapabilirsiniz. Son oturumlarınızdan bazıları aşağıda listelenmiştir; ancak, bu liste kapsamlı olmayabilir. Hesabınızın ele geçirildiğini düşünüyorsanız, şifrenizi de güncellemelisiniz.",
     "If necessary, you may logout of all of your other browser sessions across all of your devices. Some of your recent sessions are listed below; however, this list may not be exhaustive. If you feel your account has been compromised, you should also update your password.": "Gerekirse, tüm cihazlarınızdaki diğer tüm tarayıcı oturumlarınızdan çıkış yapabilirsiniz. Son oturumlarınızın bazıları aşağıda listelenmiştir; ancak bu liste kapsamlı olmayabilir. Hesabınızın ele geçirildiğini düşünüyorsanız, şifrenizi de güncellemelisiniz.",
-    "If you already have an account, you may accept this invitation by clicking the button below:": "If you already have an account, you may accept this invitation by clicking the button below:",
+    "If you already have an account, you may accept this invitation by clicking the button below:": "Zaten bir hesabınız varsa, aşağıdaki düğmeyi tıklayarak bu daveti kabul edebilirsiniz:",
     "If you did not create an account, no further action is required.": "Bir hesap oluşturmadıysanız, başka bir işlem yapmanıza gerek yoktur.",
-    "If you did not expect to receive an invitation to this team, you may discard this email.": "If you did not expect to receive an invitation to this team, you may discard this email.",
+    "If you did not expect to receive an invitation to this team, you may discard this email.": "Bu takıma davet almayı beklemiyorsanız, bu e-postayı atabilirsiniz.",
```

Comment:
`"If you did not expect to receive an invitation to this team, you may discard this email.": "Bu takıma davet almayı beklemiyorsanız, bu e-postayı yoksayabilirsiniz.",`